rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/model/PathConfigs.java

```diff
@@ -143,7 +192,8 @@ public void run() {
                         // we clear it from the list, as the path config calculation
                         // can take some time
                         changedRoots.remove(root);
-                        currentPathConfigs.replace(root, actualBuild(root));
+                        var pathConfig = actualBuild(root);
+                        currentPathConfigs.replace(root, pathConfig);
```

Comment:
why do we need this change? I guess it's an incomplete reverse of something in the PR that wasn't needed anymore?

Reply:
You are right, I will revert this change.